rootfs/etc/cont-init.d/03-config.sh

```diff
@@ -70,17 +75,20 @@ sed -e "s#@REAL_IP_FROM@#$REAL_IP_FROM#g" \
 echo "Setting Nginx XMLRPC over SCGI configuration..."
 sed -e "s!@XMLRPC_AUTHBASIC_STRING@!$XMLRPC_AUTHBASIC_STRING!g" \
   /tpls/etc/nginx/conf.d/rpc.conf > /etc/nginx/conf.d/rpc.conf
+sed -i "s!@XMLRPC_PORT@!$XMLRPC_PORT!g" /etc/nginx/conf.d/rpc.conf
```

Comment:
Would be better to chain seds:

```shell
sed -e "s!@XMLRPC_AUTHBASIC_STRING@!$XMLRPC_AUTHBASIC_STRING!g" \
  -e "s!@XMLRPC_PORT@!$XMLRPC_PORT!g" \
  /tpls/etc/nginx/conf.d/rpc.conf > /etc/nginx/conf.d/rpc.conf
```

Reply:
I did not see you could chain those. I've updated all instances.